custom_components/tuya_local/devices/friedrich_uct08b10a_airconditioner

```diff
@@ -0,0 +1,520 @@
+      # we are forced to add it so that Tuya Local gets to 100% DP match and allows us to pick this YAML for this unit
+      hidden: true
+      optional: true
+    - id: 136
+      name: dr_energy_function
+      type: string
+      # the unit's actual hardware doesn't support this functionality even though the firmware / DP is exposed
+      # we are forced to add it so that Tuya Local gets to 100% DP match and allows us to pick this YAML for this unit
+      hidden: true
+      optional: true
+
+secondary_entities:
+  - entity: switch
+    name: "Sleep Mode"
+    icon: "mdi:sleep"
+    translation_key: sleep
+    category: config
+    dps:
+      - id: 105
+        type: string
+        name: switch
+        mapping:
+            # IOT Tuya says these are the 4 values of the enum but only OFF and CHILD
+            # are supported by the firmware
+            # "off",
+            # "normal",
+            # "old",
+            # "child"
+          - dps_val: "normal"
+            value: true
+          - dps_val: "off"
+            value: false
+          - value: false
+  - entity: sensor
+    name: "Configured Temperature Unit"
+    category: diagnostic
+    class: enum
+    dps:
+      - id: 19
+        name: sensor
+        type: string
+        optional: true
+        mapping:
+          - dps_val: "f"
+            value: "°F"
+            icon: "mdi:temperature-fahrenheit"
+            icon_priority: 2
+          - dps_val: "c"
+            value: "°C"
+            icon: "mdi:temperature-celsius"
+            icon_priority: 2
+  - entity: sensor
+    translation_key: status
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 20
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: Normal
+            icon: "mdi:check-circle"
+            icon_priority: 2
+          - dps_val: E1
+            value: Fault E1
+            icon: "mdi:alert-circle"
+            icon_priority: 2
+          - dps_val: E2
+            value: Fault E2
+            icon: "mdi:alert-circle"
+            icon_priority: 2
+      - id: 122
+        name: fault2
+        type: bitfield
+        optional: true
+  - entity: binary_sensor
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 20
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - dps_val: E1
+            value: false
+          - dps_val: E2
+            value: false
+          - value: true
+      - id: 20
+        type: bitfield
+        name: fault_code
+      - id: 122
+        name: fault2
+        type: bitfield
+        optional: true
+  - entity: sensor
+    # unclear what this is, seems like intended to be total power but it's unitless per IOT and doesn't exactly track
+    # what the unit is doing so leaving as a diagnostic-only sensor that should probably be disabled by the user
+    name: totalp
```

Comment:
Not useful or unknown dps should be added as attributes of another entity rather than cluttering up the UI with additional sensors.